sql/testdata/toutiao.sql

```diff
@@ -0,0 +1,110 @@
+DROP TABLE IF EXISTS toutiao.train_processed;
```

Comment:
I see this file is named toutiao. Is this from ByteDance Inc.?

Reply:
It's from an open source dataset: https://github.com/fate233/toutiao-text-classfication-dataset

Reply:
How about we rename `toutiao` into `text`? It would be even better if we use English text dataset.

Reply:
Sure. Will add English dataset in next PR.